pom.xml

```diff
@@ -66,140 +66,144 @@
         
         <!-- 3rd party library versions -->
-        <guava.version>32.1.2-jre</guava.version>
-        <checker-qual.version>3.39.0</checker-qual.version>
-        <error_prone_annotations.version>2.22.0</error_prone_annotations.version>
+        <guava.version>33.4.0-jre</guava.version>
+        <checker-qual.version>3.48.4</checker-qual.version>
+        <error_prone_annotations.version>2.36.0</error_prone_annotations.version>
         <j2objc-annotations.version>1.3</j2objc-annotations.version>
         
-        <commons-lang3.version>3.15.0</commons-lang3.version>
-        <commons-codec.version>1.16.0</commons-codec.version>
+        <commons-lang3.version>3.17.0</commons-lang3.version>
+        <commons-codec.version>1.17.2</commons-codec.version>
         <commons-math3.version>3.6.1</commons-math3.version>
-        <caffeine.version>2.9.3</caffeine.version>
-        <transmittable-thread-local.version>2.14.2</transmittable-thread-local.version>
-        <java-util.version>2.4.0</java-util.version>
+        <commons-compress.version>1.27.1</commons-compress.version>
+        <commons-configuration2.version>2.11.0</commons-configuration2.version>
+        <commons-text.version>1.13.0</commons-text.version>
+        <caffeine.version>3.2.0</caffeine.version>
+        <transmittable-thread-local.version>2.14.5</transmittable-thread-local.version>
+        <java-util.version>3.0.0</java-util.version>
         
-        <antlr4.version>4.10.1</antlr4.version>
-        <snakeyaml.version>2.2</snakeyaml.version>
-        <gson.version>2.10.1</gson.version>
-        <jackson.version>2.16.1</jackson.version>
+        <antlr4.version>4.13.2</antlr4.version>
+        <snakeyaml.version>2.3</snakeyaml.version>
+        <gson.version>2.11.0</gson.version>
+        <jackson.version>2.18.2</jackson.version>
         <json-path.version>2.9.0</json-path.version>
-        <json-smart.version>2.4.10</json-smart.version>
-        <accessors-smart.version>2.4.9</accessors-smart.version>
-        <asm.version>9.3</asm.version>
-        <groovy.version>4.0.22</groovy.version>
-        <freemarker.version>2.3.31</freemarker.version>
+        <json-smart.version>2.5.1</json-smart.version>
+        <accessors-smart.version>2.5.1</accessors-smart.version>
+        <asm.version>9.7.1</asm.version>
+        <groovy.version>4.0.24</groovy.version>
+        <freemarker.version>2.3.34</freemarker.version>
         <bytebuddy.version>1.15.4</bytebuddy.version>
         
         <jakarta.jakartaee-bom.version>8.0.0</jakarta.jakartaee-bom.version>
         <glassfish-jaxb.version>2.3.9</glassfish-jaxb.version>
         
         <atomikos.version>6.0.0</atomikos.version>
-        <narayana.version>5.12.7.Final</narayana.version>
-        <jboss-transaction-spi.version>7.6.1.Final</jboss-transaction-spi.version>
-        <jboss-logging.version>3.2.1.Final</jboss-logging.version>
+        <narayana.version>5.13.1.Final</narayana.version>
```

Comment:
- Isn't the last version of narayana  supporting `Jakarta EE 8` 5.12.7.Final? Why is this change needed?

Reply:
This is not a jdk8 issue, this is an issue related to https://github.com/apache/shardingsphere/pull/27976. You cannot just update the Jakarta EE version without changing the java package.